apps/web/src/components/marketing/cards.tsx

```diff
@@ -20,12 +20,12 @@ export function Cards(props: Feature) {
           return (
             <li key={i}>
               <p className="flex flex-col">
-                <p>
+                <span>
```

Comment:
@AmoabaKelvin, fyi: with RSC you get a hydration error when including a `p` inside of a `p`.